setup.py

```diff
@@ -17,7 +17,8 @@
     include_package_data=True,
     zip_safe=False,
     install_requires=[
-        'apache-libcloud==1.5'
+        'apache-libcloud==1.5',
+        'ckanapi'
```

Comment:
Looks good, just needs to be pinned to a specific major version.

Reply:
f58042b As far as I can tell, `LocalCKAN.action.resource_show` (the only call you're using) should work as written with any released version. Lets keep the version range wide for maximum compatibility when installed alongside other plugins.